saleor/graphql/order/schema.py

```diff
@@ -106,7 +109,11 @@ class OrderQueries(graphene.ObjectType):
         description="Look up an order by ID or external reference.",
         id=graphene.Argument(graphene.ID, description="ID of an order."),
         external_reference=graphene.Argument(
-            graphene.String, description=f"External ID of an order. {ADDED_IN_310}"
+            graphene.String,
+            description=(
+                f"External ID of an order. {ADDED_IN_310}."
+                "\n\nRequires one of the following permissions: MANAGE_ORDERS."
```

Comment:
Shouldn't we add info when this permission is required?

Reply:
@fowczarek  What do you mean? Using externalRefernce always require manage_orders -  as added in description 🤔 .